plugins/woocommerce/tests/e2e-pw/tests/merchant/order-emails.spec.js

```diff
@@ -39,7 +39,12 @@ test.describe( 'Merchant > Order Action emails received', () => {
 		while (
 			await page.locator( '#bulk-action-selector-top' ).isVisible()
 		) {
-			await page.locator( '#cb-select-all-1' ).check();
+			// In WP 6.3, label intercepts check action. Need to force.
+			await page
+				.getByLabel( 'Select All' )
+				.first()
+				.check( { force: true } );
+
```

Comment:
Playwright UI suggested using the locator `#cb label` to check the box. But I think we can make the locator to be more user-facing by using `getByLabel` instead.